cloud/endagaweb/forms/dashboard_forms.py

```diff
@@ -153,6 +153,19 @@ def __init__(self, *args, **kwargs):
 
 class ChangePasswordForm(PasswordChangeForm):
```

Comment:
Built in validators here: https://docs.djangoproject.com/en/dev/topics/auth/passwords/#password-validation
Do you really need to override all of these fields to show a message? The validators should print their own policy text in a ValidationError on form submission.